plugins/scm.js

```diff
@@ -21,6 +20,12 @@ const token = models.user.base.extract('token').required();
 const type = core.scm.hook.extract('type').required();
 const username = models.user.base.extract('username').required();
 
+const githubStatus = [
```

Comment:
is this is specific to github, what happens for gitlab?

Reply:
That variable should be renaned https://github.com/screwdriver-cd/data-schema/pull/450#pullrequestreview-688969233

Idea is that API/Model should deal with only success/pending/failure as supported SCM status checks values. 
meta staus checks & build status should align with these values before being sent to SCM methods (updateCommitStatus). Then it's up to the individual SCM's to map it to values which pertains to it's individual SCM API values.

Reply:
Yes, renamed to `scmStatus`.

We've already mapped 'failed' state with FAILURE in Gitlab's methods. 

Please refer to this:
https://github.com/screwdriver-cd/scm-gitlab/pull/43

Reply:
So we're keeping track of this map in data-schema, models, and each scm-plugin;
but the one in data-schema is specific to github?

Reply:
I think it's better to have proper separation of concerns

Reply:
I feel schema should be the place where we define things, but it should not be made specific to Github. Also keeping track in multiple places has a high potential of issues. We already are using schema in all places and should refer to status schema from here, instead of const strings everywhere